packages/app/src/app/pages/CLI/index.tsx

```diff
@@ -6,8 +6,16 @@ import { useSignals, useStore } from 'app/store';
 
 import { Container } from './elements';
 import Prompt from './Prompt';
+import { User } from '@codesandbox/common/lib/types';
 
-const CLI = () => {
+interface Props {
+  authToken: string;
+  error: string;
+  isLoadingCLI: boolean;
+  user: User;
+}
+
+const CLI = ({ authToken, error, isLoadingCLI, user }: Props) => {
```

Comment:
@SaraVieira This can be deleted when you rebase onto 01032cb

Reply:
Oh!!! I fucked up the rebase :(

On it!